go.mod

```diff
@@ -9,10 +9,10 @@ require (
 	github.com/aws/aws-sdk-go-v2/config v1.29.9
 	github.com/aws/aws-sdk-go-v2/service/sts v1.33.17
 	github.com/coreos/go-oidc/v3 v3.12.0
-	github.com/envoyproxy/gateway v1.3.0
-	github.com/envoyproxy/go-control-plane/envoy v1.32.4
+	github.com/envoyproxy/gateway v0.5.0-rc.1.0.20250305132535-64b753bfa5fd
```

Comment:
` v0.5.0-rc` prefix is really weird but that's the last tag cut from their main branch so 🤷‍♂️